tests/spring_util_bls_test.py

```diff
@@ -61,14 +60,13 @@ def test_create_pop_from_file():
     pop = results["Proof of Possession"]
 
     # save private key to a file
-    private_key_file = "tmp_key_file_ulkdx1x42pjx"
-    with open(private_key_file, 'w+') as file:
-        file.write(private_key)
+    with tempfile.NamedTemporaryFile(mode="w+") as private_key_file:
+        private_key_file.write(private_key)
+        private_key_file.flush()
 
-    # use the private key file to create POP
-    rslts = Utils.processSpringUtilCmd("bls create pop --file {}".format(private_key_file), "create pop from command line", silentErrors=False)
-    os.remove(private_key_file)
-    results = get_results(rslts)
+        # use the private key file to create POP
+        rslts = Utils.processSpringUtilCmd("bls create pop --file {}".format(private_key_file.name), "create pop from command line", silentErrors=False)
+        results = get_results(rslts)
```

Comment:
Is the indentation change here on purpose?

Reply:
Line 68 uses `private_key_file`. Line 69 doesn't have to be inside `with` but doesn't hurt.